dace/frontend/python/newast.py

```diff
@@ -1171,6 +1173,8 @@ def parse_program(self, program: ast.FunctionDef, is_tasklet: bool = False):
         else:
             for stmt in program.body:
                 self.visit_TopLevel(stmt)
+                if isinstance(stmt, ast.Return):
+                    break
```

Comment:
is this a form of dead-code elimination? needs a comment either way

Reply:
This is unecessary, since this case is handled in preprocessing. Removed.